### DISCUSSION TITLE
V7 Beta

### DISCUSSION THREAD

**Discussion by marcus-j-davies** [zwave-js/node-red-contrib-zwave-js#202] [Q&A]
Hi @fabnavigator,

Assuming you're still with us?
we are progressing to a point a beta is being discussed.

Are you willing to join the beta test?

we are looking at V7  for our next release, and as mentioned is nearing a beta trial.

here is the change log (may change in due course)

https://github.com/zwave-js/node-red-contrib-zwave-js/blob/7.0.0/CHANGELOG.md

**Comment by fabnavigator** [comment]
Hi @marcus-j-davies,

You bet.

**Comment by marcus-j-davies** [comment]
Hi @fabnavigator  BETA 1 of V7 is up!

If you do wish to participate in the BETA:

1) Read The Change Log here before joining the trial -> https://github.com/zwave-js/node-red-contrib-zwave-js/blob/7.0.0/CHANGELOG.md
2) Do not message me directly - Please raise issues on the PR here -> https://github.com/zwave-js/node-red-contrib-zwave-js/pull/204 I will only respond on this PR - please keep your issues to the point and precise, extra noise will be a distraction.
3) Realise this is a BETA so there maybe initial issues.
4) You cannot go back a version

If your fine with that, and understand the above.

1) **BACK UP** your system (specifically **~/.node-red/zwave-js-cache**)
2) Stop Node RED
3) Run this
```
cd ~/.node-red
npm install node-red-contrib-zwave-js@beta --force
```
4) Start Node RED
5) Wait for a good few minutes (network size depending) - ZWJS V9 has updated the storage engines, and needs to convert over files

**Reply by marcus-j-davies** [reply]
New UI Guide.

![image](https://user-images.githubusercontent.com/55892693/165997487-05c5f77f-80eb-4559-b868-ac48ef898db9.png)
